src/chrome/content/rules/Comodo.xml

```diff
@@ -56,7 +56,7 @@
 	- trustlogo.com/trustlogo/javascript/cot.js (+others from /trustlogo/)
 
 -->
-<ruleset name="Comodo">
```

Comment:
This is probably over-broad. Can you confirm which domains in this file had problems?